<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessLinkDialogBase</class>
 <widget class="QDialog" name="AccessLinkDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Access Link</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pfad zur ASCII Datei welche die Vektorobjekt-Id enthält</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditIdFile">
         <property name="text">
          <string>/tmp/updated_file.txt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonIdFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pfad zur LOCK Datei, diese sollte sich im gleichen Verzeichnis befinden wie die Vektorobjekt-Id Datei</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditLockFile">
         <property name="text">
          <string>/tmp/LOCK.log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonLockFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pfad zum Visual Basic Script, welches die Verbindung zwischen QGIS und MS-ACCESS herstellt</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEditBatchFile">
         <property name="text">
          <string>/usr/bin/bash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonBatchFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name des Layers dessen Vektorobjekte angezegt werden sollen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditVectorLayer">
       <property name="text">
        <string>ALKIS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Name der Attributspalte des Layers dessen Vektorobjekte angezegt werden sollen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAttributeColumn">
       <property name="text">
        <string>id</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Zeit in Sekunden zu der die ASCII Datei auf neue Id's geprüft werden soll.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPollTime">
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonRestartPolling">
       <property name="text">
        <string>Restart Polling</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStopPolling">
       <property name="text">
        <string>Stop Polling</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReload">
       <property name="text">
        <string>Reload</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Übernehmen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Schließen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>released()</signal>
   <receiver>AccessLinkDialogBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
